### PR TITLE
Add Chinese CRNN support

### PR DESCRIPTION
**Pull request by hqkate**
It's still under development, keep improving the generalization.

1. config file for CH CRNN
2. chinese dict.txt
3. ReadMe

Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/rec/crnn/README.md
Comment (SamitHuang) [inline configs/rec/crnn/README.md:1]: dataset preparation for Chinese benchmark is missing.

File: configs/rec/crnn/crnn_resnet34_ch.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      - DecodeImage: 
+          img_mode: BGR
+          to_float32: False
+      - RecCTCLabelEncode:
+          max_text_len: *max_text_len 
+          character_dict_path: *character_dict_path
+          use_space_char: *use_space_char
+          lower: False
+      - RecResizeImg: # different from paddle (paddle converts image from HWC to CHW and rescale to [-1, 1] after resize. 
+          image_shape: [32, 100] # H, W
```
Comment (zhtmike) [inline configs/rec/crnn/crnn_resnet34_ch.yaml:93]: I remember it is [32, 320]?

Comment (hqkate) [reply configs/rec/crnn/crnn_resnet34_ch.yaml:93]: According to the experiments, [32, 100] got a better results than [32, 320]. So in this preliminary version I put [32, 100] instead. Will tune the params for [32, 320] setting later.

File: configs/rec/crnn/README_CN.md [hunk 2]
```diff
@@ -354,6 +355,23 @@ Mindocr内置了一部分字典，均放在了 `mindocr/utils/dict/` 位置，
...
+
+| 语言 | 数据说明 |
+| :------: | :------: |
+| Chinese | [中文识别数据集](../../../docs/cn/datasets/chinese_text_recognition_CN.md) | 
+
+### 预训练模型
+
+| 模型 | 语言 | 骨干网络 | 配置文件 | 模型权重下载 | 
+| :-----: | :-----: | :-----: | :-----: | :-----: |
+| CRNN | 中文 | ResNet34_vd | [crnn_resnet34_ch.yaml](https://github.com/mindspore-lab/mindocr/blob/main/configs/rec/crnn/crnn_resnet34_ch.yaml) | - |
```
Comment (SamitHuang) [inline configs/rec/crnn/README_CN.md:372]: 请上传权重并更新链接。

**Comment by zhtmike** [comment]
修改在#298， 已merge

**Pull request closed without merging**